packages/fluentui/react-northstar/package.json

```diff
@@ -49,7 +49,8 @@
     "qs": "^6.8.0",
     "react": "16.8.6",
     "react-dom": "16.8.6",
-    "simulant": "^0.2.2"
+    "simulant": "^0.2.2",
+    "tslib": "^2.1.0"
```

Comment:
```suggestion
```

This package uses Babel and its helpers, we should not depend on `tslib`

Reply:
Got it, in that case I think `importHelpers` should be removed from `scripts/typescript/tsconfig.fluentui.json`. That option technically requires `tslib` to be present, and typescript may have gotten more strict about flagging that in this version (I don't remember since I made this change a couple months ago during my first attempt).